### ISSUE TITLE
nodes.md should be added to localization files

### ISSUE THREAD

**Issue by bitcoiners** [greymass/eos-voter#145]
Currently there is only English version available. 
https://github.com/greymass/eos-voter/blob/master/nodes.md

**Comment by aaroncox** [comment]
Good call. We need to get both the `README.md` and `nodes.md` files included in our crowdin translation pipeline.
